### PR TITLE
[zh] sync /releases/supported-releases

### PR DESCRIPTION
**Pull request by asa3311**
sync /releases/supported-releases

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by my-git9 — APPROVED** [review]
LGTM

File: content/zh/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -70,10 +70,9 @@ Istio 不保证超出支持窗口期的 Minor 版本都有已知的 CVE 补丁
 
 | Minor 版本 | 没有已知 CVE 的补丁版本                        |
 | ---------- | ------------------------------------------ |
-| 1.18.x     | 1.18.0                                     |
-| 1.17.x     | 1.17.2+                                    |
-| 1.16.x     | 1.16.4+                                    |
-| 1.15.x     | 1.15.7 - 生命周期结束。不会针对新的 CVE 打补丁。 |
+| 1.18.x           | 1.18.1+                                              |
```
Comment (windsonsea) [inline content/zh/docs/releases/supported-releases/index.md:73]:
```suggestion
| 1.18.x      | 1.18.1+                                         |
```
可以 follow 上面的竖线对齐额

Comment (asa3311) [reply content/zh/docs/releases/supported-releases/index.md]: ok

**Review by windsonsea — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**